main/build.gradle

```diff
@@ -9,7 +9,7 @@ dependencies {
     compile project(":freki-core")
     compile project(":freki-cassandra")
 
-    compile project(":idmanager:freki-assign")
+    compile project(":label-manager:freki-create")
```

Comment:
Might be the package that's the problem. There's a bunch of `idmanager` references left though.